server/src/test/java/com/objectcomputing/checkins/services/guild/GuildControllerTest.java

```diff
@@ -110,13 +132,15 @@ void testEmailSentToGuildLeadWhenGuildMembersRemoved() {
         final HttpRequest<GuildUpdateDTO> request = HttpRequest.PUT("/", requestBody).basicAuth(memberProfileOfAdmin.getWorkEmail(), ADMIN_ROLE);
         client.toBlocking().exchange(request, GuildResponseDTO.class);
 
-        verify(emailSender).sendEmail(null, null,
-                "Membership Changes have been made to the Ninja guild",
-                "<h3>Changes have been made to the Ninja guild.</h3><h4>The following members have been removed:</h4><ul><li>Bill Charles</li></ul><a href=\"https://checkins.objectcomputing.com/guilds\">Click here</a> to view the changes in the Check-Ins app.",
-                "billm@objectcomputing.com"
+        assertEquals(2, emailSender.events.size());
+        assertEquals(List.of(
+                        List.of("SEND_EMAIL", "null", "null", "Membership Changes have been made to the Ninja guild", "<h3>Bill Charles has left the Ninja guild.</h3><a href=\"https://checkins.objectcomputing.com/guilds\">Click here</a> to view the changes in the Check-Ins app.", memberProfile.getWorkEmail()),
+                        List.of("SEND_EMAIL", "null", "null", "Membership Changes have been made to the Ninja guild", "<h3>Changes have been made to the Ninja guild.</h3><h4>The following members have been removed:</h4><ul><li>Bill Charles</li></ul><a href=\"https://checkins.objectcomputing.com/guilds\">Click here</a> to view the changes in the Check-Ins app.", memberProfile.getWorkEmail())
+                ),
+                emailSender.events
         );
```

Comment:
And here... I believe this is too many emails being sent 🤔

Reply:
Raised https://github.com/objectcomputing/check-ins/issues/2549

Reply:
potential PR to fix is in https://github.com/objectcomputing/check-ins/pull/2551

If that gets merged, we need to remove one of the Lists in this assert (and reduce the size assertion to 1)